src/Io/Sender.php

```diff
@@ -122,6 +122,8 @@ public function send(Request $request)
                     ));
                 }
             });
+
+            $deferred->progress(array('responseStream' => $response, 'requestStream' => $requestStream));
```

Comment:
nit: have constants defined on `Sender` instead of these strings? (Part of public API.)